py-polars/polars/dataframe/frame.py

```diff
@@ -5306,7 +5306,7 @@ def with_row_count(self, name: str = "row_nr", offset: int = 0) -> Self:
         Add a column at index 0 that counts the rows.
 
         .. deprecated::
-            Use `meth`:with_row_index` instead.
+            Use `with_row_index` instead.
```

Comment:
I think it should be
```
:meth:`with_row_index`
```
does it look alright if you make that change?

Reply:
Yea that should be it.